### ISSUE TITLE
BatchNormalization keras implementation is broken

### ISSUE THREAD

**Issue by HitLuca** [HitLuca/GANs_for_spiking_time_series#1]
For anyone checking out this repo, I would recommend to remove the BatchNormalization layers in the models as it could lead to better results. Please check my GAN collection repository for insights on how to make the models perform better https://github.com/HitLuca/GAN-collection
